tests/helpers/dataset.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+import pytest
+import string
+import random
+import re
+
+# Function or test to ensure random names of datasets
+    # Is need the hosts to call the shell in every test and for some cases of
+    # long datasets names that will generate problems with jcl the hlq size can
+    # change
+def get_dataset(hosts, hlq_size=8):
```

Comment:
Couple of thoughts about this function.

1- get_dataset name is just too general for me, would prefer "get_random_data_set_name" or "get_tmp_ds_name", or something similar that indicates to me when reading this, that the output will be a temporary data set name that most likely will be a unique name.

2- Better use data set as opposed to dataset.

Reply:
Forgot to mention, Use a docstring (https://peps.python.org/pep-0257/) for this function, this will be used a lot and having those details you commented above the function will be pretty useful.

Reply:
Resolved